htdocs/accountancy/bookkeeping/list.php

```diff
@@ -86,6 +86,7 @@
 $search_credit = GETPOST('search_credit', 'alpha');
 $search_ledger_code = GETPOST('search_ledger_code', 'alpha');
 $search_lettering_code = GETPOST('search_lettering_code', 'alpha');
+$search_not_lettering = GETPOST('search_lettering_option', 'alpha');
```

Comment:
Is "lettering" is the english term for "lettrage".
"Not Checked" and "Checked" for "Lettré" and "Non Lettré" seems better.
Another solution seems to be
"Not confirmed" and "Confirmed" 
What do you think ?

Reply:
@eldy Thanks for your advice
After some search, I find the term "Reconcile / Unreconcile", maybe use "Not reconcile" ?

Reply:
Reconcile is "Rapprochement" in french. It is often used to check you bank account compared to your bank receipt. It seems it can also be for accounting to check lines, even it is a different thing. I also found "confirmed" or "validated" and "checked".
So i suggest to use "reconcile" also for accounting line. We will have to modify a little bit translation for the current "reconciliation" feature into "bank reconciliation" so we can still make a difference between the 2 features.
Can you rename var in PR with that
$search_not_reconciled ...

Reply:
Done @eldy